src/main/java/AllCommands/DeleteCommand.java

```diff
@@ -0,0 +1,24 @@
+package AllCommands;
+
+import Support.TaskList;
+
+public class DeleteCommand {
+    public static String line;
+    public DeleteCommand(String line) {
+        DeleteCommand.line = line;
+    }
+
+    public static void deleteCommandMethod(TaskList tasks) {
+        System.out.println("____________________________________________________________");
```

Comment:
I see that you make use of the line separator multiple times, perhaps considering assigning this to a variable and referencing it, to minimize use of magic literals